corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -0,0 +1,80 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div
+  class="modal fade define-case-type-modal"
+  tabindex="-1"
+  role="dialog"
+  id="define-case-type-modal"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header" style="border-bottom: 1px solid lightgray;">
+        <button
+          type="button"
+          class="close"
+          data-dismiss="modal"
+          aria-label="Close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">
+          <i class="fa-solid fa-list-check"></i> {% trans "Case List" %}
+        </h4>
+      </div>
+      <div class="modal-body">
+        <h4>
+          <strong
+            >{% trans "Please define a case type for your case list." %}</strong
+          >
```

Comment:
Does this need the strong tag, as header tags should have their own emphasis? Should this be an h4? If staying as is, the formatting of this does seem strange

Reply:
We may also want to use more standardized versions of having emphasized text

Reply:
99811abeca3892a314ac5ea282961f1046c9affe